api/authorization-api-1_0.md

```diff
@@ -1194,8 +1194,14 @@ access_evaluation_endpoint:
 access_evaluations_endpoint:
 : OPTIONAL. URL of Policy Decision Point Access Evaluations API endpoint
 
-search_endpoint:
-: OPTIONAL. URL of Policy Decision Point Search API endpoint
+search_endpoint_subject:
```

Comment:
I would call these `search_subject_endpoint`, etc. This follows the naming of `access_evaluation_endpoint` etc.